reana-ui/src/components/Notification.module.scss

```diff
@@ -0,0 +1,15 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+@import '../styles/palette';
+
+.container {
+  margin-top: 2em;
+}
```

Comment:
Works nicely :+1: Two possible improvements:

- Please add "X" close icon so that users can close the notification window.
- Consider fading out the notification window after e.g. 10 seconds?  Hmm, perhaps it is good to keep it up "until closed by the user" for now...

Reply:
Added "X" close button as well as autoclose feature which will fade the message away after 16 seconds.

Also, refactored some more user "actions" to catch and display error messages in the top. So now it will work with following user requests:
- load config
- request token
- fetch workflows

Reply:
Thanks, fading out and closure both work nicely.  

One cosmetic comment about styling:

- on detailed record page, we have:

![2020-09-14-140613_727x95_scrot](https://user-images.githubusercontent.com/517546/93083914-99d94580-f693-11ea-98bc-a6678087203c.png)


- on the token request page, we have:

![2020-09-14-140643_722x105_scrot](https://user-images.githubusercontent.com/517546/93083936-a198ea00-f693-11ea-94ac-0f1b2511ba18.png)

Shall we perhaps harmonise the colouring and closure icons?

Reply:
Changed the workflow details page to use the new notification component with the same colors. Also, introduced "closable" property in the notification component to stay in the page as in workflow details page case.